Version 1.0.0 release preparations (#277)
Black-Kamelia/Jellyfish: README.md

[![Jenkins Tests](https://shields.io/jenkins/tests?jobUrl=https%3A%2F%2Fci.black-kamelia.com%2Fjob%2FHedera%2Fjob%2FHedera%2Fjob%2Fmaster%2F&label=Tests)
](https://ci.black-kamelia.com/job/Hedera/job/Hedera/job/develop/lastBuild/testReport/)
[![Jenkins Coverage](https://shields.io/jenkins/coverage/apiv4?jobUrl=https%3A%2F%2Fci.black-kamelia.com%2Fjob%2FHedera%2Fjob%2FHedera%2Fjob%2Fmaster%2F&label=Coverage)
](https://ci.black-kamelia.com/job/Hedera/job/Hedera/job/develop/lastBuild/coverage/)

</div>

## 🤔 What is it?

> [!IMPORTANT]
> Hedera is currently in development phase. 
> While it is stable and production-ready, some features are still missing.
> We are working hard to bring you the best experience possible.

**Hedera** is an open-source web application built with **[Ktor](https://ktor.io/)** and **[Nuxt 3](https://nuxt.com/)**.
Hedera allows you to host your files and share them. 
You can manage your account, browse your files using powerful filters, 
and share your files using a unique link or a custom one.

## ✨ Features

- **User accounts**<br>
  You can create multiple accounts, or let users sign up.

>   Each screenshot uploaded is bound to its user account.

Reviewer: each uploaded screenshot is ...

- **Disk space quota**<br>
  You can set a disk quota not to exceed; this way you can upload files without worrying about stuffing your disk completely.
  You can choose to set a custom quota for each user, or allow unlimited storage.
- **Files visibility**<br>
  Uploaded a screenshot you would like to keep private?

>   No problem, change this one's visibility, and you're done.

Reviewer: No problem: simply update its visibilty [setting], and you're done !

[![Jenkins Tests](https://shields.io/jenkins/tests?jobUrl=https%3A%2F%2Fci.black-kamelia.com%2Fjob%2FHedera%2Fjob%2FHedera%2Fjob%2Fmaster%2F&label=Tests)
](https://ci.black-kamelia.com/job/Hedera/job/Hedera/job/develop/lastBuild/testReport/)
[![Jenkins Coverage](https://shields.io/jenkins/coverage/apiv4?jobUrl=https%3A%2F%2Fci.black-kamelia.com%2Fjob%2FHedera%2Fjob%2FHedera%2Fjob%2Fmaster%2F&label=Coverage)
](https://ci.black-kamelia.com/job/Hedera/job/Hedera/job/develop/lastBuild/coverage/)

</div>

## 🤔 What is it?

> [!IMPORTANT]
> Hedera is currently in development phase. 
> While it is stable and production-ready, some features are still missing.
> We are working hard to bring you the best experience possible.

**Hedera** is an open-source web application built with **[Ktor](https://ktor.io/)** and **[Nuxt 3](https://nuxt.com/)**.
Hedera allows you to host your files and share them. 
You can manage your account, browse your files using powerful filters, 
and share your files using a unique link or a custom one.

## ✨ Features

- **User accounts**<br>
  You can create multiple accounts, or let users sign up.

>   Each uploaded screenshot is bound to a user account.

Reviewer: ```suggestion
  Each file that is uploaded is bound to a single user account.
```

- **Disk space quota**<br>
  You can set a disk quota not to exceed; this way you can upload files without worrying about stuffing your disk completely.
  You can choose to set a custom quota for each user, or allow unlimited storage.
- **Files visibility**<br>

>   Uploaded a screenshot you would like to keep private?

Reviewer: ```suggestion
  Uploaded a file you would like to keep private?
```

### Full build

> To build a complete production-ready JAR, run the package task.

Reviewer: ```suggestion
To build a complete production-ready JAR, run the build task.
```
